### PR TITLE
Add package-level godoc to pkg/kubelet/cm

### PR DESCRIPTION
**Pull request by matte21**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Add file doc.go with some rudimentary information to package kubelet/cm. This will make it easier for people approaching the kubelet codebase for the first time to quickly understand what's in the package, since its name is abbreviated and hostile to newcomers.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/cm/doc.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cm (abbreviation of "container manager") and its subpackages contain all the kubelet code
+// to manage containers. For example, they contain functions to configure containers' cgroups,
+// ensure containers run with the desired QoS, and much more functionality.
```
Comment (ffromani) [inline pkg/kubelet/cm/doc.go:19]: I'd change the "and much more functionality" in "allocate compute resources like cpus, memory, devices".

**Review by ffromani — COMMENTED** [review]
Hi, thanks for the contribution! I have a suggestion about phrasing change, other than that LGTM

**Comment by ffromani** [comment]
/retest-required

flake

File: pkg/kubelet/cm/doc.go (hunk 1 above)
Comment (matte21) [reply pkg/kubelet/cm/doc.go]: The force push to [e6652a6](https://github.com/kubernetes/kubernetes/pull/120713/commits/e6652a68caa3b16a23795cb2e789047efa8d565d) follows your suggestion, and rebases on `master`.

Comment (matte21) [reply pkg/kubelet/cm/doc.go]: cc @ffromani

**Comment by matte21** [comment]
The latest force push rebases on `master`.

**Review by mrunalp — APPROVED** [review]

**Pull request merged**